src/Appwrite/Event/Delete.php

```diff
@@ -128,6 +129,7 @@ public function trigger(): string|bool
             'resource' => $this->resource,
             'datetime' => $this->datetime,
             'hourlyUsageRetentionDatetime' => $this->hourlyUsageRetentionDatetime,
+            'function' => $this->function
```

Comment:
why do we need to pass the function here? we typically delete other resources by passing the document as part of the `$this->document` property

Reply:
Very likely left-over. I cant see where we would be setting it. Removing.